Sources/NIOSOCKS/Channel Handlers/SOCKSServerHandshakeHandler.swift

```diff
@@ -69,48 +69,46 @@ public final class SOCKSServerHandshakeHandler: ChannelDuplexHandler, RemovableC
     }
     
     public func write(context: ChannelHandlerContext, data: NIOAny, promise: EventLoopPromise<Void>?) {
+        let message = self.unwrapOutboundIn(data)
+        let outboundBuffer: ByteBuffer
```

Comment:
Nit: this variable is used entirely within the scope of `do`, and so can be defined there.